tests/Test.ADAL.Common/NativeMethodCalls.cs

```diff
@@ -207,17 +207,17 @@ public struct WINDOWPLACEMENT
         /// <returns></returns>
         [DllImport("user32", CharSet = CharSet.Ansi, SetLastError = true, ExactSpelling = true)]
         [return: MarshalAs(UnmanagedType.Bool)]
-        public static extern bool EnumChildWindows(IntPtr hWndParent, EnumChildWindowsProc lpEnumFunc, IntPtr lParam);
+        internal static extern bool EnumChildWindows(IntPtr hWndParent, EnumChildWindowsProc lpEnumFunc, IntPtr lParam);
 
         [DllImport("user32", CharSet = CharSet.Auto, ExactSpelling = true)]
         [return: MarshalAs(UnmanagedType.Bool)]
-        public static extern bool EnumThreadWindows(int threadId, EnumChildWindowsProc lpEnumFunc, IntPtr lParam);
+        internal static extern bool EnumThreadWindows(int threadId, EnumChildWindowsProc lpEnumFunc, IntPtr lParam);
                 
-        [DllImport("user32", EntryPoint = "GetClassNameA", CharSet = CharSet.Ansi, SetLastError = true, ExactSpelling = true)]
-        public static extern int GetClassName(IntPtr handleToWindow, StringBuilder className, int maxClassNameLength);
+        [DllImport("user32", EntryPoint = "GetClassNameA", CharSet = CharSet.Unicode, SetLastError = true, ExactSpelling = true)]
```

Comment:
I'm pretty sure "GetClassNameA" is going to be an Ansi function... I would expect the UTF16 version to be named "GetClassNameW"